wave_vision/include/wave/vision/tracker/tracker.hpp

```diff
@@ -0,0 +1,171 @@
+/**
+ * @file
+ * Brute force matcher implementation, derived from descriptor matcher base.
+ * @ingroup vision
+ */
+#ifndef WAVE_VISION_TRACKER_HPP
+#define WAVE_VISION_TRACKER_HPP
+
+#include <chrono>
+#include <string>
+#include <vector>
+
+#include <opencv2/opencv.hpp>
+
+#include "wave/containers/landmark_measurement.hpp"
+#include "wave/containers/landmark_measurement_container.hpp"
+#include "wave/utils/utils.hpp"
+#include "wave/vision/utils.hpp"
+
+namespace wave {
+/** @addtogroup vision
+ *  @{ */
+
+class FeatureTrack {
+ public:
+    // Constructor
+    FeatureTrack() {}
+
+    // Initializer list constructor
+    FeatureTrack(size_t id,
+                 std::vector<LandmarkMeasurement<int>> measurements,
+                 size_t first_image,
+                 size_t last_image)
+        : id(id),
+          measurements(measurements),
+          first_image(first_image),
+          last_image(last_image) {}
+
+    // Destructor
+    ~FeatureTrack() = default;
+
+    /** Returns size of the measurement vector.
+     *
+     * @return size of the measurement vector
+     */
+    size_t size() const {
+        return this->measurements.size();
+    }
+
+    /** The id of the feature */
+    size_t id;
+
+    /** All measurements of the feature */
+    std::vector<LandmarkMeasurement<int>> measurements;
+
+    /** The first image the feature is seen in */
+    size_t first_image;
```

Comment:
Yes you're right, I was mistaken.  (And that's why it would be confusing to use size_t for everything) 
Yes add an alias, especially since you're adding a field to LandmarkMeasurement